### ISSUE TITLE
Site Design

### ISSUE THREAD

**Issue by Ethan-Arrowood** [nodejs/undici-website#1]
Let's use this thread to collaborate on the site design. I've created a rough mockup in TLDraw for the idea I have in mind. I'm also part-way through implementing this in my in-progress work. Feedback and ideas appreciated!

<img width="1642" alt="Untitled" src="https://github.com/nodejs/undici-website/assets/16144158/9d3b5a04-51e9-4758-b2c3-dd6e5519acfe">

**Comment by mertcanaltin** [comment]
I would love to help @Ethan-Arrowood

**Comment by AugustinMauroy** [comment]
Is “scroll to top” necessary?

**Comment by Ethan-Arrowood** [comment]
Yes I believe it would be helpful considering how long the docs can become. Maybe less necessary considering the menu designs though 🤔

**Comment by AugustinMauroy** [comment]
Hello 👋 !
i had started create a design based on nodejs.org.
[Here the design](https://www.figma.com/design/trGCX2p47zkqEVy5V5Z2XS/Node.js-%2B-undici?node-id=12201-2413&t=KpFEQXITQNNGXqc4-1)

**Comment by Ethan-Arrowood** [comment]
Looking good!

**Comment by mertcanaltin** [comment]
looks great 🚀 

<img width="1005" alt="Screenshot 2024-07-20 at 11 51 37" src="https://github.com/user-attachments/assets/76c865f0-a984-4d57-80ad-0e430def15cd">

**Comment by AugustinMauroy** [comment]
what should the footer contain?
Also what do you want on home page

**Comment by Ethan-Arrowood** [comment]
It probably needs the regular stuff an OpenJS Foundation project should have. Probably a bunch of links? I'd use the Node.js site as a baseline

**Comment by AugustinMauroy** [comment]
we can close this one since we use doc-kit
